onnx/onnxifi.h

```diff
@@ -47,16 +47,6 @@ extern "C" {
   #endif
 #endif
 
-#if defined(ONNXIFI_LIBRARY_SUFFIX)
-#define ONNXIFI_SYMBOL_CONCAT_(prefix, suffix) prefix##suffix
```

Comment:
Why do we remove suffix?

Reply:
Suffix was needed to avoid symbol conflict where several ONNXIFI implementations are loaded with `RTLD_GLOBAL` within the same process. As we are no longer using `RTLD_GLOBAL`, and suffix didn't have any other purpose, its best to remove it to simplify loader, wrapper, implementations and users of ONNXIFI.